src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAndFilesAccordion.tsx

```diff
@@ -76,75 +76,79 @@ const ReleaseDataAndFilesAccordion = ({
             )}
           </div>
 
-          <div className="govuk-grid-row">
+          <hr className="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+
+          <div className="govuk-grid-row dfe-flex dfe-align-items--center">
             <div className="govuk-grid-column-three-quarters">
               <h3>Open data</h3>
-              <p>
-                The open data files contain all data used in this release in a
-                machine readable format.
+              <p className="govuk-!-margin-bottom-0">
+                Browse and download individual open data files from this release
+                in our data catalogue
               </p>
 
-              {!renderDataCatalogueLink && dataFiles.length > 0 && (
-                <ul className="govuk-list" data-testid="data-files">
-                  {dataFiles.map(file => (
-                    <li key={file.id}>
-                      {renderDownloadLink(file)}
-                      {` (${file.extension}, ${file.size})`}
-                    </li>
-                  ))}
-                </ul>
-              )}
-
-              {release.hasDataGuidance && (
-                <p>
-                  Learn more about the data files used in this release using our{' '}
-                  {renderDataGuidanceLink}.
-                </p>
+              {showDownloadFilesList && dataFiles.length > 0 && (
+                <Details
+                  summary="Download files"
+                  className="govuk-!-margin-bottom-0 govuk-!-margin-top-2"
+                >
+                  <ul className="govuk-list" data-testid="data-files">
+                    {dataFiles.map(file => (
+                      <li key={file.id}>
+                        {renderDownloadLink(file)}
+                        {` (${file.extension}, ${file.size})`}
+                      </li>
+                    ))}
+                  </ul>
+                </Details>
               )}
             </div>
-          </div>
-          {renderDataCatalogueLink && (
-            <div className="govuk-grid-row">
-              <div className="govuk-grid-column-three-quarters">
-                <p>
-                  You can browse and download individual open data files from
-                  this release in our data catalogue.
-                </p>
-              </div>
-              <div className="govuk-grid-column-one-quarter">
-                {renderDataCatalogueLink}
-              </div>
+            <div className="govuk-grid-column-one-quarter">
+              {renderDataCatalogueLink}
             </div>
-          )}
+          </div>
+          <hr className="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
```

Comment:
Would be good to pull out a common `SectionBreak` component with configurable `size` and `visible` props (this should probably default to true).

We should replace the various instances of this within this accordion.